Lab_Management/PowerShell/Quota/Get-LabServicesCapacityQuotas.ps1

```diff
@@ -0,0 +1,104 @@
+﻿<#
+The MIT License (MIT)
+Copyright (c) Microsoft Corporation  
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.  
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE. 
+
+.SYNOPSIS
+This script checks all the quotas for Lab Services across all regions
+
+.PARAMETER OutputCSV
+If you would like more data produced into a CSV file, provide the filename.
+#>
+
+param
+(
+    [Parameter(Mandatory=$false, HelpMessage="If you would like more data produced into a CSV file, provide the filename.")]
+    [string] $OutputCSV
```

Comment:
Is data appended to or overwrites data in the file?

Reply:
Data isn't either appended or overwritten - if the file exists we'll just print an error to the user.